server/services/participants/participant-entries.ts

```diff
@@ -282,22 +275,7 @@ export const getParticipants = async (
   return {
     data: participants.map((item) => {
       let participant = {
-        id: item.id,
-        program: item.program,
-        educationalRequirements: item.educationalRequirements,
-        firstName: item.firstName,
-        lastName: item.lastName,
-        postalCodeFsa: item.postalCodeFsa,
-        indigenous: item.indigenous,
-        driverLicense: item.driverLicense,
-        experienceWithMentalHealthOrSubstanceUse: item.experienceWithMentalHealthOrSubstanceUse,
-        preferredLocation: item.preferredLocation,
-        currentOrMostRecentIndustry: item.currentOrMostRecentIndustry,
-        roleInvolvesMentalHealthOrSubstanceUse: item.roleInvolvesMentalHealthOrSubstanceUse,
-        nonHCAP: item.nonHCAP,
-        userUpdatedAt: item.userUpdatedAt,
-        callbackStatus: item.callbackStatus,
-        distance: item.distance,
+        ..._.pick(item, user.isMoH ? participantFieldsForSuper : participantFieldsForEmployer),
```

Comment:
should this be `participantFieldsForMoH`?

Reply:
I missed your comment. Just fixed it.